includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -157,6 +161,18 @@ public function register_routes() {
 						'description' => __( 'A CSS hex color value representing the secondary branding color for this account.', 'woocommerce-payments' ),
 						'type'        => 'string',
 					],
+					'deposit_schedule_interval'         => [
+						'description' => __( 'An interval for deposit scheduling.', 'woocommerce-payments' ),
+						'type'        => 'string',
+					],
+					'deposit_schedule_weekly_anchor'    => [
+						'description' => __( 'Weekly anchor for deposit scheduling when interval is set to weekly', 'woocommerce-payments' ),
+						'type'        => 'string',
+					],
+					'deposit_schedule_monthly_anchor'   => [
+						'description' => __( 'Monthly anchor for deposit scheduling when interval is set to monthly', 'woocommerce-payments' ),
+						'type'        => 'string',
```

Comment:
Should the type be `integer`? I believe we're sending days of the number here, e.g. `1`, `2`, `3` ... `31`, etc.

Reply:
I've left as `string` as empty values are sent through as an empty string rather than null which triggers an error.